tools/src/wyvern/target/corewyvernIL/type/RefinementType.java

```diff
@@ -129,6 +184,17 @@ public ValueType doAvoid(String varName, TypeContext ctx, int depth) {
         }
     }
 
+    private static GenericArgument doAvoidArgument(String varName, TypeContext ctx, int depth, GenericArgument ga) {
```

Comment:
This method seems to belong to the `GenericArgument` class instead. Please move it there.